irispy/new_sji.py

```diff
@@ -101,11 +103,14 @@ def __repr__(self):
         #Conversion of the end date of OBS
         endobs = self.meta.get("ENDOBS", None)
         endobs = endobs.isoformat() if endobs else None
-        #Conversion of the instance start of OBS
-        instance_start = self.extra_coords["TIME"]["value"][0]
+        #Conversion of the instance start and end of OBS
```

Comment:
Just so you know, the convention of comments is to use a space between the # and the comment.

Reply:
Of course, I just forget to add a space.